cosmovisor/RELEASE_NOTES.md

```diff
@@ -3,10 +3,12 @@
 ### New execution model
```

Comment:
This was recently update and does not comply with `MD001`. Should I fix that here too?